docs/install_and_run.rst

```diff
@@ -79,14 +104,14 @@ Once the computer is connected to the admin system and activated, you
 may deploy OS2display to it.
 
 In the admin system, we have introduced three global scripts with the
-prefix "OS2displayOS".
+prefix "OS2displayPC".
 
-The first is called "OS2DisplayOS  - Installer Chromium" and will
+The first is called "OS2DisplayPC  - Installer Chromium" and will
```

Comment:
`OS2displayPC`. (Sorry, the [branding guidelines](https://os2.eu/side/os2-identitet) have eaten into my brain -- I did a bit too much graphics design in high school...)

Reply:
:+1: Will fix.